homeassistant/components/discovery/__init__.py

```diff
@@ -69,7 +69,7 @@
     SERVICE_HASSIO: ('hassio', None),
     SERVICE_AXIS: ('axis', None),
     SERVICE_APPLE_TV: ('apple_tv', None),
-    SERVICE_ENIGMA2: ('enigma2', None),
+    SERVICE_ENIGMA2: ('media_player', 'enigma2'),
```

Comment:
With this in place, we can remove the code in the enigma2 component. Just keep the docstring.

Reply:
Oh interesting. So you mean everything in the __init__.py ? (Apart from docstring like you said).

Reply:
@MartinHjelmare that is done now.